content/policyfile.md

```diff
@@ -60,17 +56,14 @@ cookbook or role from being applied immediately to all machines.
 
 ### Code Visibility
 
-When running Chef without Policyfile, the exact set of cookbooks that
+When running Chef Infra without Policyfile, the exact set of cookbooks that
```

Comment:
```suggestion
When running Chef Infra without a Policyfile, the exact set of cookbooks that
```

There are a few examples like this and it's hard to tell if it's intentional. Is it meant to be used without an article like this?